src/dali_executor/pipeline_pool.h

```diff
@@ -67,7 +67,7 @@ struct hash<triton::backend::dali::PipelineDescr> {
   size_t operator()(
       triton::backend::dali::PipelineDescr const& pk) const noexcept
   {
-    return pk.serialized_pipeline_hash ^ static_cast<size_t>(pk.batch_size);
+    return pk.serialized_pipeline_hash ^ static_cast<size_t>(pk.max_batch_size);
```

Comment:
```suggestion
    return pk.serialized_pipeline_hash + static_cast<size_t>(pk.max_batch_size);
```
+ is better than XOR, because XORing with the same number multiple times just flips between two values.

Reply:
done